client/src/app/pages/dashboard/maps/map-info/map-info.component.ts

```diff
@@ -131,6 +145,37 @@ export class MapInfoComponent implements OnInit, OnDestroy {
     }
   }
 
+  editNotificationSettings() {
+    this.dialogService.open(MapNotifyEditComponent, {
+      context: {
+        flags: (this.mapNotify ? this.mapNotify.notifyOn : 0),
+      },
+    }).onClose.subscribe(resp => {
+      if (resp) {
+        if (resp.newFlags === 0) {
+          if (this.mapNotify != null) {
+            this.locUserService.disableMapNotify(this.map.id).subscribe(() => {
+              this.mapNotify.notifyOn = 0;
+              this.mapNotifications = false;
+            }, err => {
+              this.toastService.danger('Could not disable notifications', err.message);
+            });
+          }
+        } else {
+          this.locUserService.updateMapNotify(this.map.id, resp.newFlags).subscribe(res => {
+            this.mapNotifications = true;
+            if (this.mapNotify == null)
+              this.mapNotify = res;
+            else
+              this.mapNotify.notifyOn = resp.newFlags;
+          }, err => {
```

Comment:
Try to wrap every lambda parameter in parenthesis:
```suggestion
          }, (err) => {
```